cypress/integration/LanguageChangesBasedOnBrowserDetection.spec.js

```diff
@@ -0,0 +1,22 @@
+import stubLanguages from "../support/stubLanguages"
+
+describe('User can view the website in their language', () => {
```

Comment:
I would like to see a scenario for the English language aswell

Reply:
Of course!